nautobot/utilities/tables.py

```diff
@@ -319,3 +333,39 @@ def render(self, value):
             trunc = Truncator(value)
             value = trunc.words(self.truncate_words)
         return value
+
+
+class CustomFieldColumn(tables.Column):
+    """
+    Display custom fields in the appropriate format.
+    """
+
+    def __init__(self, customfield, *args, **kwargs):
+        self.customfield = customfield
+        kwargs["accessor"] = Accessor(f"custom_field_data__{customfield.name}")
```

Comment:
Trying this out, I got an error when clicking on a custom field column header to sort the table based on that column - the below change appears to be the necessary fix:

```suggestion
        kwargs["accessor"] = Accessor(f"_custom_field_data__{customfield.name}")
```